measuring/questions/adoption-trend.md

```diff
@@ -0,0 +1,12 @@
+[⬑ back to the overall graph](../use_gqm.md)
+
+# **Question:** What is the InnerSource adoption trend?
+
+This question is something an InnerSource program asks while fostering the adoption of InnerSource.
```

Comment:
```suggestion
```
Is this redundant? 

As ISPOs, do we assume we always foster adoption of InnerSource?

Reply:
Hi @jeffabailey , the idea is that ISPOs may focus on different types of tasks overtime, e.g., fostering adoption vs raising the maturity of existing adopters. The trend of adoption is one information that can help the teams prioritize one over the other.